### PR TITLE
update packages to sxmo 1.5.0.1

### PR DESCRIPTION
**Pull request by dni**
update to sxmo 1.5.0.1

### CODE REVIEW

File: PKGBUILDS/sxmo/sxmo-xdm-config/PKGBUILD [hunk 1]
```diff
@@ -1,7 +1,7 @@
 # Maintainer: dni <office@dnilabs.com>
 pkgname=sxmo-xdm-config
-pkgver=0.3.1
-pkgrel=2
+pkgver=0.3.2
```
Comment (Danct12) [inline PKGBUILDS/sxmo/sxmo-xdm-config/PKGBUILD:3]: How does this update anything? No files has been changed.

File: PKGBUILDS/sxmo/sxmo-xdm-config/PKGBUILD [hunk 2]
```diff
@@ -1,7 +1,7 @@
...
+pkgrel=0
```
Comment (Danct12) [inline PKGBUILDS/sxmo/sxmo-xdm-config/PKGBUILD:4]: New version will always have pkgrel as 1.

File: PKGBUILDS/sxmo/sxmo-st/PKGBUILD [hunk 3]
```diff
@@ -1,7 +1,7 @@
 # Maintainer: dni <office@dnilabs.com>
 pkgname=sxmo-st
-pkgver=0.8.4
-pkgrel=1
+pkgver=0.8.4.1
+pkgrel=0
```
Comment (Danct12) [inline PKGBUILDS/sxmo/sxmo-st/PKGBUILD:4]: This too.

File: PKGBUILDS/sxmo/pn/PKGBUILD [hunk 4]
```diff
@@ -0,0 +1,39 @@
...
+	fi
+	cmake -B build \
+		-DCMAKE_INSTALL_PREFIX=/usr \
+		-DCMAKE_INSTALL_LIBDIR=lib \
+		-DBUILD_SHARED_LIBS=True \
+		-DCMAKE_BUILD_TYPE=None \
+		-DCMAKE_CXX_FLAGS="$CXXFLAGS" \
+		-DCMAKE_C_FLAGS="$CFLAGS" \
+		$CMAKE_CROSSOPTS
+	cmake --build build
```
Comment (Danct12) [inline PKGBUILDS/sxmo/pn/PKGBUILD:26]: Please use spaces instead of tabs.

File: PKGBUILDS/sxmo/pn/PKGBUILD [hunk 5]
```diff
@@ -0,0 +1,39 @@
+# Maintainer: dni <office@dnilabs.com>
+pkgname=pn
+pkgver=0.9.0
+pkgrel=0
```
Comment (Danct12) [inline PKGBUILDS/sxmo/pn/PKGBUILD:4]: pkgrel=1

File: PKGBUILDS/sxmo/mnc/PKGBUILD [hunk 6]
```diff
@@ -0,0 +1,24 @@
...
+  cd "$pkgname-$pkgver"
+	export GOFLAGS="$GOFLAGS -modcacherw"
+	go build mnc.go
+	strip mnc
+}
+
+package() {
+  cd "$pkgname-$pkgver"
+	mkdir -p "$pkgdir"/usr/bin
+	install mnc "$pkgdir"/usr/bin
```
Comment (Danct12) [inline PKGBUILDS/sxmo/mnc/PKGBUILD:22]: Ditto.

File: PKGBUILDS/sxmo/mnc/PKGBUILD [hunk 7]
```diff
@@ -0,0 +1,24 @@
+# Maintainer: dni <office@dnilabs.com>
+pkgname=mnc
+pkgver=0.3
+pkgrel=0
```
Comment (Danct12) [inline PKGBUILDS/sxmo/mnc/PKGBUILD:4]: Ditto.

**Comment by dni** [comment]
thanks for being this exact :) i think i catched everything

**Comment by Danct12** [comment]
Sorry for taking too long, I've merged it now: https://github.com/dreemurrs-embedded/Pine64-Arch/compare/d59ad25773df...206a793a2c6f

**Pull request closed without merging**